goal_src/engine/gfx/sky/sky-tng.gc

```diff
@@ -168,15 +168,24 @@
   (sky-make-sun-data *sky-parms* 1 arg0)
   (sky-make-moon-data *sky-parms* arg0)
   (let ((v1-0 *sky-tng-data*))
-   (+! (-> v1-0 off-s-0) 16)
-   (+! (-> v1-0 off-t-0) 32)
-   (+! (-> v1-0 off-s-1) -21)
-   (+! (-> v1-0 off-t-1) 42)
-   (set! (-> v1-0 time) arg0)
+    (cond ((eq? (-> *setting-control* current video-mode) '150fps)
+           (+! (-> v1-0 off-s-0) 6.4)
+           (+! (-> v1-0 off-t-0) 12.8)
+           (+! (-> v1-0 off-s-1) -8.4)
+           (+! (-> v1-0 off-t-1) 16.8)
+           )
+          (else
+           (+! (-> v1-0 off-s-0) 16)
```

Comment:
Interesting... I guess the sky moved slower in 50 fps?

Reply:
It must. These values took a while to find lol, i was just changing random sky stuff until it changed speeds